test/c/test_version_info.c

```diff
@@ -0,0 +1,24 @@
+#include "common.h"
```

Comment:
This is missing a copyright and license header. This is pointing out a gap in our CI that we're not enforcing this on the C test files. 

```suggestion
// This code is part of Qiskit.
//
// (C) Copyright IBM 2025.
//
// This code is licensed under the Apache License, Version 2.0. You may
// obtain a copy of this license in the LICENSE.txt file in the root directory
// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
//
// Any modifications or derivative works of this code must retain this
// copyright notice, and modified files need to carry a notice indicating
// that they have been altered from the originals.

#include "common.h"
```